docs/installation.md

```diff
@@ -225,6 +225,12 @@ To fix this, run the command `export PYTHONNOUSERSITE=1` before running MultiQC.
 This variable [tells Python](https://docs.python.org/3/using/cmdline.html#envvar-PYTHONNOUSERSITE)
 not to add site-packages to the system path when loading, which should avoid the conflicts.
 
+## Using the nix flake
+
+In a nix installation with [flakes](https://nixos.wiki/wiki/Flakes) enabled,
```

Comment:
Nix is not limited to NixOS, it can be installed on pretty much any *nix machine including MacOS, but they don't really have a good homepage for just the package manager. Closest I could find is the [Downloads](https://nixos.org/download.html#download-nixm).

```suggestion
In a [nix](https://nixos.org/download.html#download-nixm) installation with [flakes](https://nixos.wiki/wiki/Flakes) enabled,
```

Maybe this would be better?

Reply:
Ah ok, I didn't realise the distinction. Maybe we can tweak a bit more as I suspect quite a lot of people won't know what nix is..

How about:

```suggestion
If you're using the [nix package manager](https://nixos.org/download.html#download-nixm) with [flakes](https://nixos.wiki/wiki/Flakes) enabled, you can
```

Reply:
This looks good to me!